posthog/api/test/test_decide.py

```diff
@@ -55,31 +55,6 @@ def test_defaults_to_v2_if_conflicting_parameters(self):
         )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
-    def test_user_on_own_site_enabled(self):
-        user = self.organization.members.first()
-        user.toolbar_mode = "toolbar"
-        user.save()
-
-        self.team.app_urls = ["https://example.com/maybesubdomain"]
-        self.team.save()
-        response = self.client.get("/decide/", HTTP_ORIGIN="https://example.com").json()
-        self.assertEqual(response["isAuthenticated"], True)
-        self.assertEqual(response["supportedCompression"], ["gzip", "gzip-js", "lz64"])
-        self.assertEqual(response["editorParams"]["toolbarVersion"], "toolbar")
-
-    def test_user_on_own_site_disabled(self):
-        user = self.organization.members.first()
-        user.toolbar_mode = "disabled"
-        user.save()
-
-        self.team.app_urls = ["https://example.com/maybesubdomain"]
-        self.team.save()
-
-        # Make sure the endpoint works with and without the trailing slash
-        response = self.client.get("/decide", HTTP_ORIGIN="https://example.com").json()
-        self.assertEqual(response["isAuthenticated"], True)
-        self.assertIsNone(response["editorParams"].get("toolbarVersion"))
```

Comment:
Why were those tests removed? It seems they signaled an actual regression.

Reply:
(said in slack but) we purposefully removed this as some people complained the toolbar always being there was annoying + it slowed /decide down. You can still launch toolbar from the app. @tiina303 we should highlight this change in the release